docs/utilities/data_classes.md

```diff
@@ -495,6 +496,54 @@ In this example, we also use the new Logger `correlation_id` and built-in `corre
     --8<-- "examples/event_sources/src/bedrock_agent_event.py"
     ```
 
+### CloudFormation Custom Resource
+
+=== "app.py"
```

Comment:
Please move this example to the `examples/event_sources/src` folder and include it here. This way we can be sure that it will execute linter, typing check and others against this example.